games-emulation/mame/mame-0.264.recipe

```diff
@@ -15,29 +15,21 @@ COPYRIGHT="1997-2023 MAMEDev and contributors"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://github.com/mamedev/mame/archive/mame${portVersion/./}.tar.gz"
-CHECKSUM_SHA256="2f380a7a9344711c667aef6014d522dd876db4c04f15dbab8d14bd3b2a0d4c8c"
+CHECKSUM_SHA256="3d9f69ed3ef7c1628d5714c8ae2695ea77b1a652a93347b2703f7c862299376e"
 SOURCE_FILENAME="mame-$portVersion.tar.gz"
 SOURCE_DIR="mame-mame${portVersion/./}"
 PATCHES="mame-$portVersion.patchset"
 ADDITIONAL_FILES="
 	mame.sh
 	mame.rdef.in"
 
-ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="all !x86_gcc2 !x86"
```

Comment:
You could still keep the secondary around and disable it there :)
```diff
-ARCHITECTURES="all !x86_gcc2 !x86"
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="!x86"
```
We don't provide primary x86, so it's no use declaring it there. :)